docs/propertyestimator.rst

```diff
@@ -20,15 +20,15 @@ processes on a cluster:
 
 .. code-block:: python
 
-    estimator = PropertyEstimator(nworkers=10) # NOTE: multiple backends will be supported in the future
+    estimator = PropertyEstimatorClient(nworkers=10) # NOTE: multiple backends will be supported in the future
     computed_properties = estimator.computeProperties(dataset, parameter_sets)
 
 Here, ``dataset`` is a ``PhysicalPropertyDataset`` or subclass, and ``parameter_sets`` is a list containing
 ``ForceField`` objects used to parameterize the physical systems in the dataset.
 
 This can be a single parameter set or multiple (usually closely related) parameter sets.
 
-``PropertyEstimator.computeProperties(...)`` returns a list of ``ComputedPhysicalProperty`` objects that provide access
+``PropertyEstimatorClient.computeProperties(...)`` returns a list of ``ComputedPhysicalProperty`` objects that provide access
```

Comment:
I can't find `computeProperties()` among the `PropertyEstimatorClient`'s methods. Is it outdated?

Reply:
Thanks for the feedback @andrrizzi! I've incorporated most of this into the latest commit.